### DISCUSSION TITLE
Inclusion des  highway=track  de faible tracktype

### DISCUSSION THREAD

**Discussion by pascalpvk** [etalab/schema-amenagements-cyclables#74] [Ideas]
Bonjour,
Aujourd'hui, dans l'export BNAC réalisé par geovelo, les pistes forestières sont facilement exclus de la BNAC. En effet, elles sont taguées sur OSM sous la forme highway=track et pour bon nombre d'entre elles, la qualité du revêtement les exclu de l'export (filtre appliqué : "tracktype" NOT IN ('grade3','grade4','grade5') )
Or les surfaces de type grade3 sont parfaitement compatibles avec les VTC, et les grades 4 avec les VTT
Cf https://wiki.openstreetmap.org/wiki/Key:tracktype pour plus de détail sur le tracktype
Ces pistes sont pourtant essentielles car elles permettent des déplacements loins des voitures

**Comment by EmmanuelRoche** [comment]
Bonjour,

Par défaut les aménagements et parcours cyclables sont censés être accessibles à tous les types de vélos, y compris vélos urbains, vélos cargos ou bien vélos de course. En incluant les grades 3 et 4, un usager qui utiliserait un calculateur d'itinéraire pourrait être renvoyé sur une voie pas accessible avec son vélo. Donc pour moi c'est tout à fait cohérent de se limiter au grade 2 dans la BNAC.
Ce qui n'empêche pas de développer des applications intégrant les grades 3 ou 4, mais pour des usages spécifique type tourisme / loisirs.

Emmanuel
